src/client/components/BBackTop.less

```diff
@@ -7,6 +7,7 @@
   height: 0px;
   bottom: 60px;
   right: 20px;
+  z-index: 1500;
```

Comment:
might want to use a variable from here - https://github.com/busyorg/busy/blob/a538b8d4f3a03c4cff15f2e5c39c507783be4d51/src/client/styles/custom.less#L95 

Or create new one.